src/signature.c

```diff
@@ -7,6 +7,7 @@
 #include <openssl/crypto.h>
 #include <openssl/engine.h>
 #include <openssl/x509.h>
+#include <openssl/macros.h>
```

Comment:
This doesn't exist in OpenSSL 1.1.1.

Reply:
Yes, I actually see this in the tests but did not have the time to fix it yesterday.

Reply:
Ok, had to rediscover that nothing but ``OPENSSL_VERSION_NUMBER`` existed before 3.0...